webpack/assets/javascripts/react_app/components/ColumnSelector/column-selector.scss

```diff
@@ -0,0 +1,13 @@
+.column-selector {
+  width: 100%;
+
+  label {
+    font-size: 16px;
+    height: 36px;
+    border-bottom-color: var(--pf-global--BorderColor--200);
+    border-top-color: var(--pf-global--BorderColor--300);
+    border-left-color: var(--pf-global--BorderColor--300);
+    border-left-width: 0.7px;
+    border-top-width: 0;
+  }
+}
```

Comment:
This doesn't seem to have any effect... I've removed the whole content of the file and it didn't change anything...